package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -121,7 +121,7 @@ def vector_of_best_fit(coordinates):
     """
     centered = coordinates - coordinates.mean(axis=0)
     Mt_M = np.matmul(centered.T, centered)
-    u, s, vh = np.linalg.linalg.svd(Mt_M)
+    u, s, vh = np.linalg.svd(Mt_M)
```

Comment:
Just checking - how backwards compatible is this? Do we need to try except an import or will this work with older versions of numpy?

Reply:
I'm pretty sure we're safe since I tested the runtime downgrade to our oldest supported version (`1.23.2`) without issue.

If you don't believe me, you could use the version switcher in the NumPy docs and it is there in version `1.13`: https://numpy.org/doc/1.13/reference/generated/numpy.linalg.svd.html

That's pretty ancient now

Reply:
So sorry @tylerjereddy - I completely missed this from your previous comment message (it's the second time in a row over two PRs really sorry about this).